src/iuse.cpp

```diff
@@ -2509,13 +2509,30 @@ int iuse::dig( player *p, item *it, bool t, const tripoint & )
     int moves;
 
     if( g->m.has_flag( "DIGGABLE", pnt ) ) {
-        if( g->m.ter( pnt ) == t_pit_shallow ) {
+        const ter_id ter = g->m.ter( pnt );
+        if( ter == t_pit_shallow ) {
             if( p->crafting_inventory().has_quality( DIG, 2 ) ) {
                 moves = MINUTES( 90 ) / it->get_quality( DIG ) * 100;
             } else {
                 p->add_msg_if_player( _( "You can't deepen this pit without a proper shovel." ) );
                 return 0;
             }
+        } else if( ter == t_grave ) {
+            if( g->u.has_trait_flag( "SPIRITUAL" ) ) {
+                add_msg( m_info, _( "You refuse to touch the sacred resting place of the dead." ) );
```

Comment:
I personally feel like it should just be a much more intense morale penalty, like human flesh and Spiritual. The vomiting should also be moved to Spiritual, personally, but that's just me.

Reply:
Maybe make it possible to dig graves (perharps even keeping the fun bonus) for those who have both Spiritual and Psychopath (getting fun from breaking a taboo)? That's how it works for cannibal morale penalty/bonus, after all.

Reply:
Both ideas are good. I'll implement them.